### PR TITLE
converting md files to rst for sphinx

### PR DESCRIPTION
**Pull request by 1sanz**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -50,6 +50,7 @@
     'sphinx.ext.napoleon',
     'sphinx.ext.autodoc',
     'sphinx_tabs.tabs',
+    'sphinx.ext.autosectionlabel',
```
Comment (flaub) [inline docs/conf.py]: Please add this new dependency into the `conda/unix.yml` file

File: docs/manual_api/cpp_tile_edsl.rst [hunk 2]
```diff
@@ -0,0 +1,1070 @@
+C++ Tile eDSL
+#############
+
+The C++ Tile eDSL (Embedded Domain Specific Language) provides developers with a
+
```
Comment (flaub) [inline docs/manual_api/cpp_tile_edsl.rst]: Remove all the extra newlines.

File: docs/manual_api/cpp_tile_edsl.rst [hunk 3]
```diff
@@ -0,0 +1,1070 @@
...
+************************
+
+Sum Over Axis
+================
+
+We're ready to look at some C++ Tile code! Here's an operation that takes the
+
+sum over axis `0` of a 2D tensor (in Keras this would be ``K.sum(I, axis=0)``):
+
+.. code-block:: c++
```
Comment (flaub) [inline docs/manual_api/cpp_tile_edsl.rst]: Use tabs to show c++ and python examples

File: docs/manual_api/cpp_tile_edsl.rst [hunk 4]
```diff
@@ -0,0 +1,1070 @@
...
+  dimensional tensor to be evaluated elementwise. `T` and `F` must have the same
+
+  shape, and unless `C` is known to be a constant at compile time, both will be
+
+  evaluated.)
+
+Types
+=====
+
+- `Tensor`: Multidimensional arrays of a fixed shape. The scope of a tensor is
```
Comment (flaub) [inline docs/manual_api/cpp_tile_edsl.rst]: Generally, we should link these refs to API objects into the API docs.

**Comment by flaub** [comment]
Also, could you remove all the old `.md` files while you're at it?

File: docs/manual_api/cpp_tile_edsl.rst [hunk 5]
```diff
@@ -0,0 +1,1070 @@
+C++ Tile eDSL
```
Comment (flaub) [inline docs/manual_api/cpp_tile_edsl.rst]: I don't think this file should go into `docs/manual_api`, perhaps into `docs/articles` or `docs/usage`.

File: docs/manual_api/index.rst [hunk 6]
```diff
@@ -7,5 +7,6 @@ API
 
    core
    edsl
+   cpp_tile_edsl
```
Comment (flaub) [inline docs/manual_api/index.rst]: I don't think this belongs here. This should be the reference guide for the API.

File: docs/usage/install.rst [hunk 7]
```diff
@@ -1,3 +1,230 @@
...
+
+Finally, set up PlaidML to use a preferred computing device:
+
+.. code-block::
+
+    plaidml-setup
+
+
+
+You can test the installation by running MobileNet in `plaidbench <https://github.com/plaidml/plaidbench>`_. Remember to
```
Comment (dgkutnic) [inline docs/usage/install.rst]: The `pkaidbench` repo has been archived and the contents are now a part of the `plaidml` repo. We should get rid of the link to the old `plaidbench` repo.

File: docs/usage/install.rst [hunk 8]
```diff
@@ -1,3 +1,230 @@
-Installation
-============
+Installation Instructions
+#########################
+
+PlaidML supports :ref:`Ubuntu`, :ref:`macOS` , and :ref:`Microsoft Windows` operating systems.
+
+Ubuntu
```
Comment (dgkutnic) [inline docs/usage/install.rst]: Can we use tabs for these installation instructions now that we have them?

File: docs/usage/install.rst [hunk 9]
```diff
@@ -1,3 +1,230 @@
...
+
+Finally, set up PlaidML to use a preferred computing device:
+
+.. code-block::
+
+    plaidml-setup
+
+PlaidML should now be installed! You can test the installation by running
+
+MobileNet in `plaidbench <https://github.com/plaidml/plaidbench>`_.
```
Comment (dgkutnic) [inline docs/usage/install.rst]: like mentioned above, please remove any links to the archived `plaidbench` repo

File: docs/manual_api/cpp_tile_edsl.rst (hunk 5 above)
Comment (dgkutnic) [reply docs/manual_api/cpp_tile_edsl.rst]: Additionally, this file should probably be renamed to `tile_edsl.rst` and any content that's not specific to a particular language should have language-specific references removed. Then we can start adding tabs for language-specific examples, and have a more generalized document overall. Thoughts?

File: docs/manual_api/cpp_tile_edsl.rst [hunk 10]
```diff
@@ -0,0 +1,1070 @@
...
+
+output tensor is inferred from the shape of the input tensor(s). In most binary
+
+operations, if the input tensors have different shapes, the output shape is
+
+determined by broadcasting together the input shapes. If this is impossible or
+
+ambiguous, it is an error.
+
+Common operations (not comprehensive; example tensor variable names provided to
```
Comment (dgkutnic) [inline docs/manual_api/cpp_tile_edsl.rst]: We may want to link to the reference API for these

File: docs/usage/contributing.rst [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+
+
+Guidelines
+**********
+
+* Create unit tests for new features and bug fixes. Integration tests are
+
+  required for larger features.
+
+* Pre-commit linters will be available soon.
```
Comment (dgkutnic) [inline docs/usage/contributing.rst]: Pre-commit linters are already available. But in general, we may need to rewrite this.

File: docs/conf.py (hunk 1 above)
Comment (1sanz) [reply docs/conf.py]: this is a builtin extension like sphinx.ext.autodoc so I believe this is not required.

File: docs/manual_api/cpp_tile_edsl.rst (hunk 2 above)
Comment (1sanz) [reply docs/manual_api/cpp_tile_edsl.rst]: the conversion tool added them. I think I got them all :)

File: docs/manual_api/cpp_tile_edsl.rst (hunk 3 above)
Comment (1sanz) [reply docs/manual_api/cpp_tile_edsl.rst]: Done

File: docs/manual_api/cpp_tile_edsl.rst (hunk 5 above)
Comment (1sanz) [reply docs/manual_api/cpp_tile_edsl.rst]: Done

File: docs/usage/install.rst (hunk 8 above)
Comment (1sanz) [reply docs/usage/install.rst]: Done

File: docs/conf.py (hunk 1 above)
Comment (flaub) [reply docs/conf.py]: OK

File: docs/usage/configuration.rst [hunk 12]
```diff
@@ -1,23 +1,14 @@
...
-  * `PLAIDML_DEVICE_IDS` - (string) the name of the device to use
-     with PlaidML (to see a list of devices, run `plaidml-setup`)
-
+  * `PLAIDML_EXPERIMENTAL` - (0 or 1) determines whether to enable experimental mode in PlaidML 
+  * `PLAIDML*DEVICE*IDS` - (string) the name of the device to use with PlaidML (to see a list of devices, run ``plaidml\-setup``)
 Below is an example of how to set the device configuration environment variables
 for PlaidML.
+.. code-block::
+  export PLAIDML_EXPERIMENTAL=1
+  export PLAIDML*DEVICE*IDS=opencl*intel*uhd*graphics*630.0
```
Comment (flaub) [inline docs/usage/configuration.rst]: These are all gone in plaidml-v1. `PLAIDML_DEVICE_IDS` is now `PLAIDML_DEVICE` and a new env var is introduced: `PLAIDML_TARGET`

**Pull request merged**